web/components/ResourceViews/DataTables/NodeDataTable.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <v-row>
+    <v-col>
+      <v-card class="ma-2" outlined>
+        <v-card-title class="mb-1">
+          Nodes <v-spacer></v-spacer>
+          <v-text-field
+            v-model="search"
+            append-icon="mdi-magnify"
+            label="Search"
+            single-line
+            hide-details
+          ></v-text-field
+        ></v-card-title>
+        <v-data-table
+          :headers="headers"
+          :items="nodes"
+          :items-per-page="5"
+          :search="search"
+          multi-sort
+          @click:row="onClick"
+        ></v-data-table>
+      </v-card>
+    </v-col>
+  </v-row>
+</template>
+
+<script lang="ts">
+import {
+  computed,
+  inject,
+  onMounted,
+  defineComponent,
+} from "@nuxtjs/composition-api";
+import NodeStoreKey from "../../StoreKey/NodeStoreKey";
+import { V1Node } from "@kubernetes/client-node";
+import PodStoreKey from "../../StoreKey/PodStoreKey";
+import {} from "../../lib/util";
+
+export default defineComponent({
+  setup() {
+    const pstore = inject(PodStoreKey);
+    if (!pstore) {
+      throw new Error(`${PodStoreKey} is not provided`);
+    }
+
+    const nstore = inject(NodeStoreKey);
+    if (!nstore) {
+      throw new Error(`${NodeStoreKey} is not provided`);
+    }
+
+    const getNodeList = async () => {
+      await nstore.fetchlist();
+    };
+
+    onMounted(getNodeList);
+
+    const nodes = computed(() => nstore.nodes);
+    const onClick = (node: V1Node) => {
+      console.log("aaa");
+      nstore.select(node, false);
+    };
+    const search = "";
+    const headers = [
```

Comment:
# Need help
We need to discuss which columns should be selected for display.
I think that displaying all parameters is not good.
The current columns of each data table are my randomly selected.

Reply:
In commit [d225fbc](https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/119/commits/d225fbcfdd17243f58278aff835786695d698215), I changed some columns(headers).

These columns are displayed in each table.
- name
- namespace
- capacity
- status(phase)
- condition
- creation-time
- update-time

And for other columns, the unique values of each resource were chosen.